samples/create-and-save-map/src/main/java/com/esri/arcgismaps/sample/createandsavemap/MainActivity.kt

```diff
@@ -23,26 +23,15 @@ import androidx.compose.material3.MaterialTheme
 import androidx.compose.material3.Surface
 import androidx.compose.runtime.Composable
 import androidx.lifecycle.viewmodel.compose.viewModel
-import com.arcgismaps.ArcGISEnvironment
-import com.esri.arcgismaps.sample.sampleslib.theme.SampleAppTheme
-import com.esri.arcgismaps.sample.createandsavemap.screens.MainScreen
 import com.arcgismaps.toolkit.authentication.DialogAuthenticator
-import com.arcgismaps.toolkit.authentication.signOut
 import com.esri.arcgismaps.sample.createandsavemap.components.MapViewModel
-import kotlinx.coroutines.runBlocking
+import com.esri.arcgismaps.sample.createandsavemap.screens.MainScreen
+import com.esri.arcgismaps.sample.sampleslib.theme.SampleAppTheme
 
 class MainActivity : ComponentActivity() {
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
-        // This sample uses an ArcGIS Online login to be able to save a map as an ArcGIS portal item
-        // No need for license strings or an API key
-        ArcGISEnvironment.apiKey = null
-
-        // Sign out of any portals which are already authenticated
-        runBlocking {
-            ArcGISEnvironment.authenticationManager.signOut()
-        }
```

Comment:
Thanks for doing this here - I can close the draft PR that's been sitting around for ages.

Reply:
Thanks for creating it, I added it here to make sure the new sample viewer design contains your PR changes.